README.md

```diff
@@ -0,0 +1,54 @@
+# Dangl.XRechnung.Examples
+
+> An online version of the converter is freely accessible at [my blog](https://blog.dangl.me/xrechnung-converter/)  TODO: ADD A WORKING LINK
+> A more detailed feature description is available [on my website](https://www.dangl-it.com/products/xrechnung/).
+
+> This project requires **Visual Studio 2017** or newer. Otherwise, the Dotnet CLI is also supported.
+
+This is a _demonstration_ library intended to show the usage of the Dangl.XRechnung project.
+To build this solution, you need to copy four packages into the `./packages` folder, relative to the root of this project:
+* Dangl.XRechnung
+* Dangl.Data.Shared
```

Comment:
That doesn't sound right😀 I think we need ´Dangl.XRechnung`, `Dangl.AVA` and `Dangl.AVA.IO`. `Dangl.Data.Shared` is on the public NuGet feed, so no need to reference it here.